__tests__/database/schema/ChatSchema.test.ts

```diff
@@ -38,6 +40,7 @@ const fromDatabaseObjectExpectation = {
   createdAt: 'createdAt',
   updatedAt: 'updatedAt',
   rawJson: {},
+  attachmentJson: [],
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Again, the `attachmentJson` field is initialized as an array, but it seems to be treated as a string in other parts of the code. Please ensure that the data type is consistent across all uses.
```diff
-   attachmentJson: [],
+   attachmentJson: '',
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->